### PR TITLE
Rename 'Recent comments' to 'Recent discussion', and move 'New shortform' button to shortform section of the frontpage

### PR DESCRIPTION
**Pull request by Will-Howard**
Screenshot 2023-05-03 at 12 36 33
Screenshot 2023-05-03 at 12 36 41

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1204522589816383) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/components/common/CommentsListCondensed.tsx [hunk 1]
```diff
@@ -36,7 +49,15 @@ const CommentsListCondensed = ({label, terms, initialLimit, itemsPerPage, showTo
 
   const showLoadMore = !loading && (count === undefined || totalCount === undefined || count < totalCount)
   return <>
-    <SectionTitle title={label} className={classes.subheader} />
+    <SectionTitle title={label} className={classes.subheader} >
+      {currentUser?.isReviewed && shortformButton && !currentUser.allCommentingDisabled && <div onClick={toggleShortformFeed}>
+        <SectionButton>
+          <AddBoxIcon />
+          New Shortform Post
```
Comment (Will-Howard) [inline packages/lesswrong/components/common/CommentsListCondensed.tsx:56]: Maybe this should just say "New" now?

Comment (jpaddison3) [reply packages/lesswrong/components/common/CommentsListCondensed.tsx:56]: I'm chill with it being overly explicit

File: packages/lesswrong/components/recentDiscussion/RecentDiscussionFeed.tsx [hunk 2]
```diff
@@ -62,14 +61,7 @@ const RecentDiscussionFeed = ({
...
       <AnalyticsInViewTracker eventProps={{inViewType: "recentDiscussion"}}>
         <SingleColumnSection>
-          <SectionTitle title={title}>
-            {currentUser?.isReviewed && shortformButton && !currentUser.allCommentingDisabled && <div onClick={toggleShortformFeed}>
-              <SectionButton>
-                <AddBoxIcon />
-                New Shortform Post
-              </SectionButton>
-            </div>}
-          </SectionTitle>
```
Comment (jpaddison3) [inline packages/lesswrong/components/recentDiscussion/RecentDiscussionFeed.tsx:72]: Forum-gate this please

**Review by jpaddison3 — APPROVED** [review]
ASTLAC

File: packages/lesswrong/components/common/CommentsListCondensed.tsx (hunk 1 above)
Comment (agnesstenlund) [reply packages/lesswrong/components/common/CommentsListCondensed.tsx:56]: or "New shortform" to help differentiate from "New post" 🤔

**Comment by agnesstenlund** [comment]
- [ ] Would reduce this gap slightly

Screenshot 2023-05-03 at 14 51 54

Also not a huge fan of our "New post/shortform" button styling but think we should change that later and ship this 🚢 

Otherwise looks great!

File: packages/lesswrong/components/recentDiscussion/RecentDiscussionFeed.tsx (hunk 2 above)
Comment (Will-Howard) [reply packages/lesswrong/components/recentDiscussion/RecentDiscussionFeed.tsx:72]: Good catch!

**Pull request merged**